.circleci/workflows.yml

```diff
@@ -54,6 +56,7 @@ linuxArm64WorkflowFilters: &linux-arm64-workflow-filters
     - equal: [ develop, << pipeline.git.branch >> ]
     # use the following branch as well to ensure that v8 snapshot cache updates are fully tested
     - equal: [ 'update-v8-snapshot-cache-on-develop', << pipeline.git.branch >> ]
+    - equal: [ 'cacie/dep/electron-27', << pipeline.git.branch >> ]
     - equal: [ 'chore/update_octokit', << pipeline.git.branch >> ]
     - equal: [ 'em/circle2', << pipeline.git.branch >> ]
```

Comment:
do you also need to add `setup_should_persist_artifacts` to publish the binary for testing?